tests/Directory.Build.props

```
@@ -15,9 +15,9 @@
     <MicrosoftNetTestSdkVersion>17.4.0</MicrosoftNetTestSdkVersion>
 		<!--GHSA-5crp-9r3c-p9vr-->
 		<NewtonsoftJsonVersion>13.0.2</NewtonsoftJsonVersion>
-		<XunitVersion>2.4.2</XunitVersion>
-		<XunitRunnerVisualStudioVersion>2.4.5</XunitRunnerVisualStudioVersion>
-    <XunitAssertVersion>2.4.2</XunitAssertVersion>
+		<XunitVersion>2.9.0</XunitVersion>
```

```suggestion
		<XunitVersion>2.9.2</XunitVersion>
```
https://xunit.net/#:~:text=Additional%20copyrights-,Current%20Releases,-Stable